tests/unit-tests/test_dataloading.py

```diff
@@ -30,10 +30,17 @@
     generate_dummy_dist_graph,
     generate_dummy_dist_graph_reconstruct,
     generate_dummy_dist_graph_homogeneous_failure_graph,
+    generate_dummy_dist_graph_multi_task,
     create_distill_data,
+    SIZE_DICT
 )
 
 import graphstorm as gs
+from graphstorm.config import (TaskInfo,
```

Comment:
Should we import EdgeRegression as well?

Reply:
EdgeRegression is not used.